Package.swift

```diff
@@ -9,7 +9,7 @@ let package = Package(
         .target(name: "SQLiteObjc"),
         .testTarget(name: "SQLiteTests", dependencies: ["SQLite"], path: "Tests/SQLiteTests")
     ],
-    swiftLanguageVersions: [4]
+    swiftLanguageVersions: [5]
```

Comment:
I think we'll want the following for CI to pass with SwiftPM.
```suggestion
    swiftLanguageVersions: [.version("5"), .v4_2, .v4]
```